### PR TITLE
[pdc-describe] update postgresql version

### LINKED ISSUE #4092: migrate to postgresql 15
upgrade databases for pdc-describe and pdc-discovery

- [ ] update pdc-discovery-staging
- [ ] update pdc_describe-staging

### PR DESCRIPTION
**Pull request by kayiwa**
upgrade postgresql version
this will close #4092

### CODE REVIEW

File: group_vars/pdc_describe/main.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-postgres_version: 13
+postgres_version: 15
```
Comment (acozine) [inline group_vars/pdc_describe/main.yml:2]: Won't this change production as well as staging? I don't think we're ready to change both.

**Review by acozine — COMMENTED** [review]
The ticket says to upgrade pdc_describe_staging and pdc_discovery_staging. Has pdc-discovery already been done? Also, do we need to have separate values for the psql version in staging and prod (see below)?

**Comment by acozine** [comment]
FYI for folks upgrading to 15 in development environments, see https://github.com/pulibrary/figgy/pull/5918#issuecomment-1660571628 for a gotcha (with fix).

File: group_vars/pdc_describe/main.yml (hunk 1 above)
Comment (kayiwa) [reply group_vars/pdc_describe/main.yml:2]: this only updates the client. This https://github.com/pulibrary/princeton_ansible/issues/4105 will document how this role functions.

**Comment by acozine** [comment]
The values got updated for pdc_discovery in #4100.

**Pull request merged**